app/models/cfe/v2/result.rb

```diff
@@ -7,16 +7,32 @@ def assessment_result
         result_hash[:assessment][:assessment_result]
       end
 
+      def capital_assessment_result
+        capital[:assessment_result]
+      end
+
+      def capital_contribution_required?
```

Comment:
Could this method use the one above?
`def capital_contribution_required?
    capital_assessment_result  == 'contribution_required'
 end`

Reply:
Good catch